src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -259,6 +259,7 @@ public static Vector<T> AsVector<T>(this Vector128<T> value)
         /// <remarks>On x86, this method corresponds to __m128i _mm_set1_epi8</remarks>
         /// <returns>A new <see cref="Vector128{Byte}" /> with all elements initialized to <paramref name="value" />.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [Intrinsic]
```

Comment:
Shouldn't this tag (and the respective handling) be on `CreateScalar`?

The `Create(T value)` methods are for broadcasting, that is setting all elements of the vector to `value`.

Reply:
Actually, I see that these are mapping to `dup` but then I'm not connecting how `fmov` and `movi` allow broadcasting the value

Reply:
`CreateScalar()` intrinsify work is being done separately by @echesakovMSFT  in https://github.com/dotnet/runtime/pull/35300. This PR focuses on `Create()` methods.

Reply:
Nevermind, I found the relevant instructions. There are a few variants of `fmov` and `movi` that I had missed.

Reply:
yeah, these are specifically "C7.2.122 FMOV (vector, immediate)" and "C7.2.197 MOVI"